docker/Dockerfile

```diff
@@ -16,21 +16,24 @@
 # cd to //maldoca
 # docker build -f /docker/Dockerfile -t <tag> .
 
-# Based on Ubuntu 18.04
-FROM ubuntu:bionic
+# Based on Ubuntu 20.04
+FROM ubuntu:focal
```

Comment:
nope, i did not double check with chrome folks. for now, i just want to get ci with linux & clang running to avoid obvious breakages when submitting stuff for the windows port. 

ubuntu 18 has clang6 as default version (which throws many warnings during compilation). ubuntu 20 supports clang 11. hence, the update.

Reply:
will submit it for now. plz ping me if u have concerns.